opencog/nlp/learn/README

```diff
@@ -253,21 +253,21 @@ So, set up the text-ingestion pipeline:
   you will want to use `zh-pre` or `yue-pre` for the language; this
   disables the addition of spaces between hanzi characters.
 
-10) After doing at least several days worth of parsing  (i.e. at least
+9) After doing at least several days worth of parsing  (i.e. at least
    hundreds of articles, tens of thousands of sentences) the word-pair
    mutual information can be extracted. Do the stuff in section "Mutual
    Information" section below.
 
    If the parsing is interrupted, you can restart the various scripts;
    they will automatically pick up where they left off.
 
-11) Using the extracted mutual information for word-pairs, the MST
+10) Using the extracted mutual information for word-pairs, the MST
    parsing step can be started. See below. Run this for several days.
    At the end of this, you will have a bunch of statistics on
    "pseudo-connector sets": pairs which associate a word to a
    disjunct of pseudo-connectors.
 
-12) Perform clustering research.  You're on your own, now.
+11) Perform clustering research.  You're on your own, now.
```

Comment:
There are now several additional steps after this.  After obtaining a collection of disjuncts, you will probably want to compute the marginals and the MI for those.  I have an assortment of half-finished, half-working scripts that attempt to consolidate or "cluster" the disjuncts -- and those require assorted addtional scripts to be run.  But its still cutting edge, I guess...

Reply:
Nice, please include instructions and scripts when you have them ready :)